demos/toolbar/waterfall-flexible-toolbar.html

```diff
@@ -80,7 +53,7 @@
         </p>
       </div>
     </main>
-    <footer>
+    <footer class="demo-toolbar-floating-footer">
```

Comment:
I think it would be more BEM-ish to be `demo-toolbar__floating-footer` or `demo-toolbar__footer--floating` if we are trying to be BEM

Reply:
Ditto above